utils/weights.py

```diff
@@ -63,6 +66,15 @@ def get_weights(time_set, staff):
 
     weights = set_weights()
 
+    # Scale preferences to number of weeks
+    weights["preferences"] = scale_weight_to_weeks(weights["preferences"], time_set["weeks"])
+
+    # Scale weights relatively
+    weights = scale_weights_relatively(weights, staff)
+
+    pprint(weights)
+    breakpoint()
```

Comment:
Skal breakpointed være med?

Reply:
Good catch! Neither breakpoint nor pprint should be included here. Will fix